.circleci/config.yml

```diff
@@ -182,7 +182,7 @@ commands:
                       command: riot -v run 'wait' << parameters.wait >>
             - run:
                 command: |
-                  riot list --hash-only '<<parameters.pattern>>' | shuf | circleci tests split | xargs -n 1 -I {} riot -v run --exitfirst --pass-env -s {} $( << pipeline.parameters.coverage >>=false; echo '--no-cov' )
+                  .scripts/ddtest riot list --hash-only '<<parameters.pattern>>' | shuf | circleci tests split | .scripts/ddtest xargs -n 1 -I {} riot -v run --exitfirst --pass-env -s {} $( << pipeline.parameters.coverage >>=false; echo '--no-cov' )
```

Comment:
I don't think we want `./scripts/ddtest` here right?

Reply:
yeah, maybe this is why we cannot use `run_test` for the integration ones.

Reply:
but we don't want to use the docker container if we don't have to, because of the overhead it adds (it'll make all others slower)

Reply:
Sounds good. I'll revert back to not using `run_test`.